test/witan/models/load_data_test.clj

```diff
@@ -0,0 +1,67 @@
+(ns witan.models.load-data-test
+  (:require [witan.models.load-data :refer :all]
+            [clojure.test :refer :all]
+            [schema.core :as s]))
+
+(defn- same-coll? [coll1 coll2]
+  (= (set coll1) (set coll2)))
```

Comment:
I think we need to return as equal two collections with the same content but possibly of different type (vector or list) and ordered differently.